### ISSUE TITLE
Two different Data augmentations?

### ISSUE THREAD

**Issue by priyarana** [GuillaumeErhard/Supervised_contrastive_loss_pytorch#1]
Hi,
According to the theory, in stage 1, two different types of augmentations should be applied to a batch of samples.
1) crop and resize
2) AutoAugment/randaugment/simaugment.
It seems the code doesn't have augmentation number (2). Are the mentioned results from only augmentation number (1) which has been applied twice to the same set of samples?

Thanks

**Comment by GuillaumeErhard** [comment]
Hello,

Yes in this repository only AutoAugment is implemented along with the first step. Because in the paper the best result came from this augmentation. But if you intend to use it on custom dataset you should rather use randaugment/simaugment as they are more generalist.

If you have the correct flag when running the training script the two step you described will be used. See snippet 

![image](https://user-images.githubusercontent.com/25333848/147348196-d3064c13-ea72-4f3c-8968-8a7861219477.png)

My result are reported in the readme according to your description like this. First column only cross-entropy with step 1  then cross-entropy with step 1 and 2 and finally Supervised constrastive loss with step 1 and two. 

This was to done to better judge the gain brought by AutoAugment or any powerful data augment and the gains brought by this new loss.
